### ISSUE TITLE
Yarn@Berry support

### ISSUE THREAD

**Issue by F0rsaken** [roots/bud#387]
* [x] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
* [x] This request isn't a duplicate of an existing issue
* [x] I've read the [docs](https://roots.io/documentation/) and followed them (if applicable)
* [x] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

## Description

[Description of the issue]
Using bud in a project that uses Yarn@Berry don't work. I received error:
```
(node:23777) [ERR_REQUIRE_ESM] Error Plugin: @roots/bud [ERR_REQUIRE_ESM]: Must use import to load ES Module: /{FOLDER_ROOT}/node_modules/@roots/bud/node_modules/globby/index.js
require() of ES modules is not supported.
require() of /{FOLDER_ROOT}/node_modules/@roots/bud/node_modules/globby/index.js from /{FOLDER_ROOT}/node_modules/@oclif/config/lib/plugin.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /{FOLDER_ROOT}/node_modules/@roots/bud/node_modules/globby/package.json.
```

I found out, that `Yarn@Berry` installs `@roots/bud` dependencies in it's folder and so there is a `globby` there. When I just use normal Yarn everything works.

## Steps to reproduce

1. Init simple project with `npm init`
2. Change `yarn`: `yarn set version berry`
3. Install `bud`: `yarn add @roots/bud @roots/bud-cli --dev`
4. run `yarn bud init`

**Expected behavior:**
Should not crash

**Actual behavior:**
Crashes `bud`

**Reproduces how often:** [What percentage of the time does it reproduce?]
everytime

## Versions

`@roots/bud: ^4.7.0-10`
`@roots/bud-cli: ^4.7.0-10`

**macOS 11.3.1**
**Yarn: 1.22.11**
**Yarn Berry: 3.0.1**
**Node: 14.17.5**

**Comment by F0rsaken** [comment]
I managed to solve my problem, by adding

```json
{
	...
	"resolutions": {
		"globby": "^11.0.0"
	}
}
```
in `package.json`, but I see this more as a workaround than a real fix

**Comment by F0rsaken** [comment]
as it turned out, installed version was `4.7.0-next.11`, event tho in `package.json` I specified `^4.7.0-10`. If `4.7.0-10` is used, there is no need for `resolutions` field

**Comment by kellymears** [comment]
That's totally my bad, @F0rsaken. I messed up the tagging on `4.7.0`; it should have been tagged `next` but it wasn't. There has been a lot of work related to publishing workflows (#180) that we're close to having locked down (so we aren't manually tagging in the future). 

Sorry for the confusion and I'm glad you got it sorted.

**Comment by ceesvanegmond** [comment]
@kellymears This issue is still there with the latest 5.0.0-next19. Adding the `resolutions` fixes this in 5.0 as well.

**Comment by kellymears** [comment]
I'm closing this issue in favor of #941, which should address the underlying cause. This isn't a problem with yarn. This is a problem with modern JS 🥲
